bim2sim/elements/bps_elements.py

```diff
@@ -907,9 +911,22 @@ def bound_shape(self):
                 shape = PyOCCTools.make_faces_from_pnts(pnts)
         shape = BRepLib_FuseEdges(shape).Shape()
 
+        if conv_required:
+            # scale newly created shape of space boundary to correct size
+            conv_factor = (1 * self.ifc_units.get('IfcLengthMeasure')).to(
```

Comment:
`self.ifc_units.get('IfcLengthMeasure')` is already called before in line 859 and willebe called again in line 927. Put it into a variable at the start and use the variable from then on.

Reply:
Fixed.